docs/docs/en/architecture/load-balance.md

```diff
@@ -49,12 +49,7 @@ Each worker has two weights parameters, weight (which remains constant after war
 This algorithm reports its own load information to the registry at regular intervals. Make decision on two main pieces of information:
 
 - load average (default is the number of CPU cores * 2)
-- available physical memory (default is 0.3, in G)
+- available physical memory (default is 70%, in percentage)
 
 If either of these is lower than the configured item, then this worker will not participate in the load. (no traffic will be allocated)
```

Comment:
We use cpu/memory/worker slot usage to calculate

Reply:
Fixed.